src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    protected ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        ColorSupplier.Color randomColor = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(100) + 1;

```

Comment:
add constant for 100
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/a6a03339-813c-4a34-9c13-7ea378d6ac04)